bin/run_controller

```diff
@@ -153,6 +153,10 @@ if [[ -z $vxlan ]]; then
     docker exec $CONTAINER ip link set data0 up
 fi
 
+if [ $CLEAN_MAC_PREFIX != "$DEFAULT_CLEAN_MAC_PREFIX" ]; then
+    docker exec $CONTAINER /root/bin/rename_site_config $CLEAN_MAC_PREFIX
```

Comment:
I think this should go inside controller_go -- basically just pass the CLEAN_MAC_PREFIX in (as per the docker_envs use above), and then inside of the controller sort out what to do about it. Generally better to avoid the "docker exec" solution because it's a lot harder to debug/diagnose/trace -- the only reason it's done for the data0 interface is because it has to be done externally after the container is created (effective race condition).